purchase_requisition_order_remaining_qty/models/purchase_requisition_type.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PurchaseRequisitionType(models.Model):
+    _inherit = "purchase.requisition.type"
+
+    quantity_copy = fields.Selection(
+        selection_add=([("remaining_qty", "Remaining Qty")])
```

Comment:
Put it on second place and using a similar redaction and avoid abbreviations on this kind of text that are not part of a column:

```suggestion
        selection_add=([("remaining_qty", "Use remaining quantities of agreement"), ("none", )])
```